src/BaGetter.Aws/S3StorageOptions.cs

```diff
@@ -19,6 +20,8 @@ public class S3StorageOptions
 
     public string Prefix { get; set; }
 
+    public Uri Endpoint { get; set; }
```

Comment:
Add `[RequiredIf]` attribute to both `Endpoint` and `Region` please.

Reply:
Tested with Linode. I will change Options as suggested.